driver/bpf/fillers.h

```diff
@@ -3876,6 +3876,8 @@ FILLER(sys_ptrace_x, true)
 		res = bpf_val_to_ring_dyn(data, 0, PT_UINT64, 0);
 		if (res != PPM_SUCCESS)
 			return res;
+
```

Comment:
do we want to remove the above `if`?

Reply:
I was 100% sure someone was going to complain about that either way, since we have both forms in the current code :-D. Will remove it.

Reply:
I know we have it the other way (without the `if`). If we have these both ways, it's up to you I guess (and let's choose a direction to drift towards ;))

Reply:
No problem, I like it better too. Just repushed. Thanks for looking.